Workbooks/Azure Active Directory/EntitlementManagementAccessPackageActivity/EntitlementManagementAccessPackageActivity.workbook

```diff
@@ -0,0 +1,68 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Entitlement management access package activity"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "700e461e-68e1-457c-b269-9772078f0458",
+            "version": "KqlParameterItem/1.0",
+            "name": "AccessPackageId",
+            "label": "Access Package",
+            "type": 2,
+            "description": "Show entitlement management audit events for the specified access package",
+            "isRequired": true,
+            "query": "AuditLogs | where TimeGenerated > ago(3653d) and Category == \"EntitlementManagement\" and Result == \"success\" | mvexpand TargetResources | extend AccessPackageId=tostring(TargetResources.id), TTYPE=TargetResources.type, TD=strcat(tostring(TargetResources.displayName),\" \",TargetResources.id) | where TTYPE == \"AccessPackage\" | project TD,AccessPackageId | distinct AccessPackageId,TD | sort by TD asc\r\n",
```

Comment:
are you sure you want this? `> ago(3653d)` ?  10 years of data in the workspace by default?  this is going to blow through the cold cache be a slow query for sure.  you might want to look into maybe swapping this and the since one and making this one depend on that?

and ideally not make the defaults the worst case slowest case?

Reply:
Makes sense, I'll take the suggestion to use the 'since' value as scope for audit events containing the access package name/id.